### DISCUSSION TITLE
How to not remove parts from the Arduino board

### DISCUSSION THREAD

**Discussion by fakkuyuu** [sanni/cartreader#729] [Need help]
Can I use trace cutting instead of removing the two parts from the Arduino board?
If it is possible please tell me where to cut
![IMG_20230309_222143](https://user-images.githubusercontent.com/120779915/224036731-2b0e7404-8b2a-424b-a3ef-6b3d583856f7.jpg)

I drew a red line to be cut on the photo. Will this work?

**Comment by sanni** [comment]
With your Arduino variant just desoldering/removing/cutting the fuse should be enough.

![224036731-2b0e7404-8b2a-424b-a3ef-6b3d583856f7](https://user-images.githubusercontent.com/19169494/224102577-75d19558-c273-428b-baa5-4e63dc428ad8.jpg)

**Reply by fakkuyuu** [reply]
![IMG_20230309_222143 (1)](https://user-images.githubusercontent.com/120779915/224206714-818d845f-515b-40e5-8cfc-74b26bb87a2a.jpg)
is this okay?

**Reply by sanni** [reply]
Yes that will work.

**Reply by fakkuyuu** [reply]
very thanks

**Reply by sanni** [reply]
You also don't need the black wire.

**Reply by fakkuyuu** [reply]
It's even more comfortable. I would try 1 cut and 1 red wire.

**Reply by fakkuyuu** [reply]
worked fine
![IMG_20230312_152132](https://user-images.githubusercontent.com/120779915/224528704-a64309e4-d6bf-404f-b7ae-035199d12bbb.jpg)
